linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/oval/fedora.xml

```diff
@@ -5,14 +5,52 @@
       <affected family="unix">
         <platform>multi_platform_fedora</platform>
       </affected>
-      <description>Look for crypto policy correctly configured in /etc/crypto-policy/config.</description>
+      <description>Look for crypto policy correctly configured in /etc/crypto-policy/config, and the policy is current.</description>
     </metadata>
-    <criteria>
+    <criteria operator="AND">
       <criterion comment="check for crypto policy correctly configured in /etc/crypto-policy/config"
       test_ref="test_configure_crypto_policy" />
+      <criterion comment="check for crypto policy correctly configured in /etc/crypto-policy/state/current"
+      test_ref="test_configure_crypto_policy_current" />
+      <criterion comment="Check if update-crypto-policies has been run after config update" test_ref="test_crypto_policies_updated" />
```

Comment:
Is this test needed? just checking time stamps isn't enough?

Reply:
yes, it is needed. What happens if you `touch /etc/crypto-policies/state/current`? Check would pass, and you could have a different policy.

Reply:
Good point.